projects/transformers/run_utils.py

```diff
@@ -693,6 +693,20 @@ def toggle_method(*args, **kwargs):
     return toggle_method
 
 
+def format_eval_results(eval_results, run, task_name):
+
+    ignore_keys = ["steps", "epoch"]
+    new_eval_results = {}
+    for key in eval_results.keys():
```

Comment:
That same approach discussed in the callback could be used to simplify this code block